app/about_us/OurValues.js

```diff
@@ -0,0 +1,45 @@
+import styles from "./page.module.css";
+export default function OurValues() {
+  return (
+    <>
+      <ol>
+        <li>
+          <h3>Exploration: </h3>{" "}
```

Comment:
If you need some extra space after the header it is better to use some css 😉 
You can create a class for the `<ol className={styles.ourValuesContainer}>` and use that class to apply some margins/paddings for the children.